gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/SwaggerService_CreateAPITest.java

```diff
@@ -525,4 +541,17 @@ public void shouldPrepareAPIFromSwaggerV3WithComplexReferences() throws IOExcept
         final SwaggerApiEntity api = prepareInline("io/gravitee/rest/api/management/service/mock/json-api.yml", true, true);
         validatePolicies(api, 2, 5, asList("/drives"));
     }
+
+    @Test
+    public void shouldPrepareAPIFromSwaggerV3WithURLAndPathMappingOnly() throws IOException {
+        final SwaggerApiEntity swaggerApiEntity = prepareUrl("io/gravitee/rest/api/management/service/openapi.yaml", false, true);
+
+        assertTrue(swaggerApiEntity.getPathMappings().containsAll(asList("/pets", "/pets/:petId")));
+        validatePathMappings(swaggerApiEntity, asList("/pets", "/pets/:petId"));
+        assertEquals(0, swaggerApiEntity.getPaths().size());
```

Comment:
Should it contain "/" ?

Reply:
do you think it is necessary that  `/` is always present ? it was not the case before. 🤔 
it was only added if there was no path mapping as a default list value but not a mandatory list item

Reply:
but I'm not sure why there is this default `/` if you have any explanation I'm up for a call 🥺 🙏